server/src/main/java/io/druid/segment/realtime/firehose/EventReceiverFirehoseFactory.java

```diff
@@ -146,12 +183,18 @@ public Response addAll(Collection<Map<String, Object>> events)
           }
         }
 
-        return Response.ok().entity(ImmutableMap.of("eventCount", events.size())).build();
+        return Response.ok(
+            objectMapper.writeValueAsString(ImmutableMap.of("eventCount", events.size())),
+            contentType
```

Comment:
I think there are a lot of other places that do not do this (add content type to the Response). Do we need to add it elsewhere?